app/javascript/components/map/VMapFilters.vue

```diff
@@ -1,52 +1,41 @@
 <template>
-  <div
-    v-if="show"
-    class="v-map-filters"
-  >
-    <v-map-header 
-      closeable
-      v-model="title"
-      @close="onClose"
-    />
+  <div v-if="show" class="v-map-filters">
+    <v-map-header closeable v-model="title" @close="onClose" />
     <div class="v-map-filters__body">
-      <v-map-pa-search />
+      <v-map-pa-search-dropdown
+        :state="convertSearchTypeToDropdownOption(searchType)"
+        :label="dropdownLabel"
+        :options="dropdownOptions"
+        @change="onDropdownChange"
+      />
+      <v-map-pa-search @search="onSearch" :type="searchType" />
       <div class="v-map-filters__overlays">
-        <div
-          v-for="(overlay, index) in overlays"
-          :key="index"
-          class="v-map-filters__overlay"
-        >
+        <div v-for="(overlay, index) in overlays" :key="index" class="v-map-filters__overlay">
```

Comment:
Can we keep the broken lines for more than one attribute? I think this is in the linter.

Reply:
You should just be able to run yarn lint on this file 🤞 Single quotes in the javascript is also set in the eslint config.

Reply:
What's the reason for single quotes in the lint config given that JS doesn't differentiate between the two types?

Reply:
Consistency and to avoid merge conflicts. No strong reason for single vs double I don't think.

Reply:
Ah I see. I've just setup ESLint on VSCode so this is going to do a lot of the work for me